src/main/java/org/gridsuite/modification/server/entities/equipment/modification/VoltageInitModificationEntity.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.modification.server.entities.equipment.modification;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.Setter;
+import org.gridsuite.modification.server.dto.VoltageInitGeneratorModificationInfos;
+import org.gridsuite.modification.server.dto.VoltageInitModificationInfos;
+import org.gridsuite.modification.server.dto.ModificationInfos;
+import org.gridsuite.modification.server.entities.ModificationEntity;
+
+import javax.persistence.CollectionTable;
+import javax.persistence.ElementCollection;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import java.util.List;
+import java.util.stream.Collectors;
+
+
+/**
+ * @author Franck Lecuyer <franck.lecuyer at rte-france.com>
+ */
+@NoArgsConstructor
+@Getter
+@Setter
+@Entity
+@Table(name = "voltageInitModification")
+public class VoltageInitModificationEntity extends ModificationEntity {
+    @ElementCollection
+    @CollectionTable(name = "voltageInitGeneratorsModification")
```

Comment:
missing index and foreign key name

note: it's actually missing pretty much everywhere I see...

Reply:
Done